### ISSUE TITLE
More Accessibility fixes.

### ISSUE THREAD

**Issue by erumaro** [erumaro/eruma-roku#9]
Running some more tests and some fixes to be made are:

- [x] - <aside> not allowed inside another landmark. Change the aside in the footer into a div.
- [x] - Project showcase on front-page has a slight transparency when not focused/hovered. This makes the text less readable.
- [x] - Landmarks: Some content might lack landmarks. Try to find a suitable landmark.
- [x] - Add an underline to Copyright notice link or make it stand out more.
- [x] - Some aria and screen reader texts aren't translated to Swedish. Add them to translation file.
- [x] - To change in Gutenberg: About page has a skipped heading level. Move existing content up one level or add the missing heading.
- [x]  - On the same page some content is bolded which gives you an alert that it might be a title. Either remove the bold or find a different solution to style it.
- [x]  - The blog comment field has a link in aksimet notice that uses a link with barely any contrast to main text and not underline. Add an underline.
- [ ]  - Project links improvements. Currently a bit generic, try to use clearer language,

**Comment by erumaro** [comment]
- [ ] - Make focus more clear for those tabbing on the pages.
- [x] - Skills section tabs up the first column and then continues without visually changing to the next column.
- [x] - The back to top button should semantically be placed last in the footer, not after article. It should be the last you can tab too.
- [x] - Link to image in the about me page. Is the link necessary? As it is now, you can tab to it.
- [ ] - Content/Texts: Swedish and English is mixed randomly on the pages, which causes screen readers to read the pages weirdly.

**Comment by erumaro** [comment]
- [x] Mobilmeny: aria-expanded is set as false when the menu is open. Change to true in js file.

**Comment by erumaro** [comment]
- [x] Group links with `<ul>` or `<ol>`. All `<nav>` content has this issue.
Note: This choice was made when the theme was created. Semantically wrapping links with only `<nav>` is ok. But a list provides extra information for screen readers about number of links so they don't have to traverse the links one by one when they are too many.

**Comment by erumaro** [comment]
Some updates.
The content fixes are live on the site.

Menu, generic links and different languages issues are on the way to be fixed. I believe the content will take the most time. I need to figure out what is used where. Some things might be used in English language apps.

**Comment by erumaro** [comment]
Readded list to nav walker. Will navigate the site with tab to spot any contrast clashes. Current version of site goes live soon.
